configure.ac

```diff
@@ -913,6 +913,10 @@ if test "$PLATFORM_INFO" != "none"; then
     AC_DEFINE_UNQUOTED([PLATFORM_INFO],["${PLATFORM_INFO}"],[OpenThread platform information])
 fi
 
+AC_SUBST(OPENTHREAD_EXAMPLES_DA15000)
```

Comment:
Intention of this commit was only to change README.md Something must go wrong with the process. I'll draw-back this pull request.